README.md

```diff
@@ -874,15 +874,7 @@ easier understanding and reading of a test.
     end
     ```
 
-* Add an example ensuring that the FactoryGirl.created model is valid.
-
-    ```ruby
-    describe Article do
-      it 'is valid with valid attributes' do
-        expect(article).to be_valid
-      end
-    end
-    ```
+* Use [FactoryGirl.lint](https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#linting-factories) to ensure that all Factories are valid.
```

Comment:
If we would like to add FactoryGirl.lint to the style-guide, it may be better under the "Factories/Fixtures" section https://github.com/reachlocal/rspec-style-guide#factoriesfixtures